state-sync/state-sync-v2/state-sync-driver/src/tests/driver.rs

```diff
@@ -33,10 +39,22 @@ use storage_service_client::StorageServiceClient;
 
 // TODO(joshlind): extend these tests to cover more functionality!
 
+#[tokio::test(flavor = "multi_thread")]
+async fn test_auto_bootstrapping() {
+    // Create a driver for a validator with a waypoint at version 0
+    let subscription_event_key = EventKey::random();
```

Comment:
nit: this piece of code is duplicated in many other newly added tests, may be create a common helper function for code reuse?

Reply:
Cleaned this up a little :) I think most instances of reuse are slightly different, so I'm inclined to leave this as it is.